GenderPayGap.WebUI/Views/AdminRemoveUser/ConfirmRemoving.cshtml

```diff
@@ -0,0 +1,115 @@
+}
+
+@section BeforeMain {
+    @{
+        var crumbs = new List<CrumbViewModel>();
+        crumbs.Add(new CrumbViewModel
+        {
+            Text = "Admin",
+            Href = Url.Action("Home", "Admin")
+        });
+        if (Model.FromViewUserPage)
+        {
+            crumbs.Add(new CrumbViewModel
+            {
+                Text = Model.UserFullName,
+                Href = Url.Action("ViewUser", "AdminViewUser", new {id = Model.UserId})
+            });
+        }
+        else
+        {
+            crumbs.Add(new CrumbViewModel
+            {
+                Text = Model.OrganisationName,
+                Href = Url.Action("ViewOrganisation", "AdminViewOrganisation", new {id = Model.OrganisationId})
+            });
+        }
+        crumbs.Add(new CrumbViewModel
+        {
+            Text = "Remove user from organisation"
+        });
+    }
+
+    @(Html.GovUkBreadcrumbs(new BreadcrumbsViewModel
+    {
+        Crumbs = crumbs
+    }))
+}
+
+<span class="govuk-caption-xl">Administration</span>
+<h1 class="govuk-heading-xl">
+    Remove a user from an organisation
+</h1>
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <form method="POST" action="@Url.Action("RemoveUserFromAnOrganisation", "AdminRemoveUser")">
+            @(Html.AntiForgeryToken())
+            @(Html.GovUkErrorSummary())
+
+            <table class="govuk-table">
+                <tbody class="govuk-table__body">
+                    <tr class="govuk-table__row">
+                        <th scope="row" class="govuk-table__header">User</th>
+                        <td class="govuk-table__cell">@(Model.UserFullName)</td>
+                    </tr>
+                    <tr class="govuk-table__row">
+                        <th scope="row" class="govuk-table__header">Organisation</th>
+                        <td class="govuk-table__cell">@(Model.OrganisationName)</td>
+                    </tr>
+                </tbody>
+            </table>
+
+            @(Html.HiddenFor(m => m.OrganisationId))
+            @(Html.HiddenFor(m => m.UserId))
+            @(Html.HiddenFor(m => m.OrganisationName))
+            @(Html.HiddenFor(m => m.UserFullName))
+            @(Html.HiddenFor(m => m.FromViewUserPage))
+
+            <div class="govuk-form-group">
+                @Html.GovUkTextAreaFor(
+                    m => m.Reason,
+                    labelOptions: new LabelViewModel
+                    {
+                        Text = "What is the reason for this change?",
+                        Classes = "govuk-label--m"
+                    },
+                    formGroupOptions: new FormGroupViewModel
+                    {
+                        Classes = "govuk-!-margin-top-6"
+                    })
+
+                @(Html.GovUkButton(new ButtonViewModel
+                {
+                    Text = "Confirm",
+                    Classes = "govuk-!-margin-bottom-4"

```

Comment:
This action does an unreversable delete, so I think we should make the button a warning style by adding `govuk-button--warning` to the Classes.

I might also make the Text be something more specific, e.g. `Remove user from organisation`